Admin/FieldDescription.php

```diff
@@ -98,6 +98,12 @@ public function getValue($object)
             $object = $this->getFieldValue($object, $parentAssociationMapping['fieldName']);
         }
 
+        // Support embedded object for mapping
+        // Ref: http://doctrine-orm.readthedocs.io/projects/doctrine-orm/en/latest/tutorials/embeddables.html
+        if (isset($this->getFieldMapping()['declaredField'])) {
```

Comment:
We have been having a problem with Travis, that's why the build wasn't red, but the next commit will probably report this.